protocol/messenger_response.go

```diff
@@ -812,3 +824,25 @@ func (r *MessengerResponse) GetUpdatedProfileShowcases() []*ProfileShowcase {
 	}
 	return showcases
 }
+
+func (r *MessengerResponse) AddSeveralSeenAndUnseenMessages(messages []*SeenUnseenMessages) {
```

Comment:
I am not sure what `adding` messages mean ?
Could you explain what are you trying to achieve please?

Reply:
This `MessengerResponse` we are using as a response type in all messenger api functions, so it's an aggregate for all possible Messenger's outputs